lms/templates/certificates/_about-edx.html

```diff
@@ -0,0 +1,10 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
You imported but don't use it. You can avoid import block or use it in the template.